Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -255,14 +255,48 @@ function Upload-FilesToAzureContainer
             $blobStorageURI = $blobStorageEndpoint+$containerName+"/"+$blobPrefix
         }
 
+        $azcopy = "AzCopy\AzCopy.exe"
+        Write-Verbose "Calling AzCopy = $azcopy" -Verbose
+
+        $sourceLocationDir = [System.IO.Path]::GetDirectoryName($sourcePath)
+        $filenm = Split-Path $sourcePath -leaf
+        $isFile = Test-Path -Path $sourcePath -PathType Leaf
+
+        $azlog = ("{0}\..\azlog" -f $sourceLocationDir)
```

Comment:
You should not be creating the log file in relative path. As said the path can be UNC.